src/vlinder/make_report.py

```diff
@@ -0,0 +1,219 @@
+"""
+This file contains two functions which make a report of the information performed in the tool.
+"""
+
+import os
+from pathlib import Path
+from datetime import date, datetime
+import shutil
+from fpdf import FPDF
+from vlinder.presentation import PDF
+import cv2
+
+
+def determine_position_images(orientation, image):
```

Comment:
Why is this one outside the class?

Reply:
Because it is static and doesn't use any information from _self_ but I placed it inside the class now